packages/authorization-process/src/services/validators.ts

```diff
@@ -73,6 +74,20 @@ export const assertOrganizationIsPurposeConsumer = (
   }
 };
 
+export const assertOrganizationIsDelegate = (
+  organizationId: TenantId,
+  purpose: Purpose,
+  delegation: Delegation
+): void => {
+  if (organizationId !== delegation.delegateId) {
```

Comment:
As done in other services, here I would perform all the checks on the delegation assuming that the delegation could be in any state, since the signature is a generic `Delegation` - even if these checks could be redundant

```suggestion
  if (    delegation.delegateId !== authData.organizationId ||
    delegation.delegatorId !== purpose.consumerId ||
delegation.eserviceId !== purpose.eserviceId ||
delegation.kind !== delegationKind.delegatedConsumer ||
delegation.state !== delegationState.active) {
```